src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/NodeService.kt

```diff
@@ -27,9 +29,17 @@ class NodeService(
   @Autowired private val transactionsEventStoreRepository: TransactionsEventStoreRepository<Any>
 ) {
   var logger: Logger = LoggerFactory.getLogger(NodeService::class.java)
+
   suspend fun closePayment(
     transactionId: UUID,
     transactionOutcome: ClosePaymentRequestV2Dto.OutcomeEnum
+  ): ClosePaymentResponseDto {
+    return closePayment(transactionId, transactionOutcome, null)
+  }
+  suspend fun closePayment(
+    transactionId: UUID,
+    transactionOutcome: ClosePaymentRequestV2Dto.OutcomeEnum,
+    authorizationCode: String?
```

Comment:
Also with the other comment I'd remove this overload and just use one with an `Optional` parameter